src/stories/10_Introduction/_AMP_Story_Hello_World.html

```diff
@@ -0,0 +1,109 @@
+<!-- 
+  ## Introduction
+
+  Learn how to create your first AMP Story. The [amp-story](https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/amp-story.md) extension provides a new format for displaying visual content. 
```

Comment:
Change link to https://www.ampproject.org/docs/reference/components/amp-story.html

Reply:
done